composer.json

```
@@ -13,19 +13,17 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
+        "php": ">=8.0",
         "predis/predis": "^1.1.1",
         "doctrine/cache": "~1.3",
-        "b1rdex/predis-compressible": "^0.5.0"
+        "b1rdex/predis-compressible": "^1.0",
+        "symfony/symfony": "~5.0 || ~6.0"
```

I don't think we should require the whole symfony framework, but instead only required components.
```suggestion
        "symfony/cache": "~5.0 || ~6.0",
        "symfony/config": "~5.0 || ~6.0",
        "symfony/dependency-injection": "~5.0 || ~6.0",
        "symfony/event-dispatcher-contracts": "~5.0 || ~6.0",
        "symfony/http-foundation": "~5.0 || ~6.0",
        "symfony/http-kernel": "~5.0 || ~6.0"
```

There is also the case of `m6web/guzzle-http-bundle` but maybe let's not make it mandatory but only in `suggest` ? I don't really know if it can have any impact being missing, or being required...

---

We do not require m6web/guzzle-http-bundle in our project and got this bundle to work fine so it's definitely not mandatory

---

Yeah I suppose your're right, it's only an opt-in class